src/fixtures/businesses.ts

```diff
@@ -0,0 +1,33 @@
+import { Business } from "../routes/commercialRegistrations/types/business";
+
+export const businesses: Business[] = [
```

Comment:
I think, it would be good to extend it with more records, otherwise we will always have one record on each search ?

Reply:
Yes: https://github.com/kontist/mock-solaris/pull/197/commits/9556898f1d039c8b3aeed07e5e1d005b774a6b2c